rust/arrow/src/compute/kernels/sort.rs

```diff
@@ -122,7 +122,7 @@ impl Default for SortOptions {
     fn default() -> Self {
         Self {
             descending: false,
-            nulls_first: false,
+            nulls_first: true,
```

Comment:
Why the change here?

Reply:
this is to match spark's default behavior, i have added a comment.

Reply:
Okay, that's fine, we could set the default in DataFusion instead, but the way you've done it is fine. I would have taken that as a lazy approach to avoid changing all the test cases :)